src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
```

Comment:
The variable name 'VALUES' is misleading. It actually contains the keys of the 'robot' object. Consider renaming it to 'KEYS' or something similar that accurately represents its contents.